editor/utils/mobile.js

```diff
@@ -0,0 +1,13 @@
+import { BREAK_MEDIUM } from '../constants';
```

Comment:
For consistency we should add the `Internal dependencies` DocBlock.

Reply:
Added 👍